frontend/src/components/table.tsx

```diff
@@ -0,0 +1,187 @@
+import React, { useState } from 'react';
+import '../styles/table.css';
+
+interface Driver {
+  name: string;
+  netid: string;
+  email: string;
+  phone: string;
+}
```

Comment:
The Driver interface will eventually need to be exactly consistent with the backend's Driver model. This can wait until designs are finished.